apps/bfd-server/dev/bfd-data-dictionary/app/dd-transformer/dd_to_csv.py

```diff
@@ -15,20 +15,19 @@ def build_csv_header():
 # TODO: refactor this func to address hardcoded field names
 def field_val(element_json, field):
     # handling varies by field
-    match field:
-        case "appliesTo" | "ccwMapping" | "cclfMapping":   # convert array to csv with ; as delimiter
-            return ";".join(element_json[field])
-        case "resource" | "element" | "derived" | "note" | "fhirPath" | "example":   # pull from fhirMapping object
-            return element_json["fhirMapping"][0][field]
-        case "discriminator" | "additional":  # pull from fhir mapping object and convert from array to csv with ; as delim
-            return ";".join(element_json["fhirMapping"][0][field])
-        case "AB2D" | "BB2" | "BCDA" | "BFD" | "DPC" | "SyntheticData": # convert from array to individual columns
-            if field in element_json["suppliedIn"]:
-                return "X"
-            else:
-                return ""
-        case _:  # default just return the value from the top level key
-            return element_json[field]
+    if field in {"appliesTo", "ccwMapping", "cclfMapping"}:
```

Comment:
Any reason this was converted to and if/else instead of a case? Seems like a functionally neutral change. (with the comments about what its doing removed in the process)

Reply:
When run on jenkins the script fails because the build has python 3.9 which doesn't have the `match` case statement.  I restored the comments that I had removed.